src/tests/fm.test.js

```diff
@@ -412,5 +418,22 @@ describe('app: hod.proving', () => {
         expect(fm.getNotFoundNino('Resource not found: RK123****', 'RK123456C', '')).toEqual('RK123456C')
       })
     })
+
+    describe('showFeedbackOptions', () => {
+
+      it('should display not passed feedback options determined by passed result', () => {
+        //if user is passed
+        var element = $compile('<fm-feedback passed="true" callback="feedbackDone" nino="applicant" ng-show="showFeedbackForm"></fm-feedback>')($rootScope);
+        $rootScope.$digest()
+        expect(element.html()).toContain("Not Passed on Cat A Salaried");
```

Comment:
I can see the logic here - we'd expect if one `Not Passed` reason is showing then all of them are, but it's an assumption.

I would be inclined to add checks for all of the options but I'll leave the choice up to you.